app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConfigurationSteps/ConfigurationSteps.tsx

```diff
@@ -29,9 +41,19 @@ const ConfigurationSteps = ({ currentStage }: AppProps) => {
 			href: '#',
 			onClick: () => { history.push('/install'); }
 		},
+		{
+			id: 'create',
+			label: 'Create server',
+			percentageComplete:
+				currentStage === 'create' || currentStage === 'install'
```

Comment:
What's the point of this?

Reply:
Why do we even need a progress indicator when it only has 2 values (0 and 100)? I don't understand the logic here, either.

Reply:
Not sure. I'll look into it and refactor it to make it clearer.

Reply:
@mboudreau The ternary here is meant to check which step you are on, and then determine whether you have completed that step or not. If you are on `Step 1: Install`, you won't have completed `Step 2: Create`, but if you are on `Step 2: Create`, you will have completed that step. Looking over this again, it's not readable to anyone who doesn't understand how the AtlasKit component works. When I originally created this component, it would not work if you added any further steps, hence the comment [here](https://github.com/atlassian/jenkins-for-jira/pull/20/files#diff-ecadc634d815960142e227324427661e44516139729669482165efb7158b8e00R3). I initially decided to push this through as we are on a time crunch, but I'm not happy with the quality of this code and will decline this PR and write it again to make it more readable.

@thombergs The progress indicator shows where you are in the 3 step process. It has 3 values but this line is determining what "progress" to show depending on which step you are on. I.e.`Should the progress line be on step 1, 2 or 3?`.

Progress line on step 3:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/28718897/180719046-b6b84432-2ac3-4d4c-85e9-f43ac2ff5d4e.png">